docs/grpc-tls-howto.md

```diff
@@ -9,6 +9,9 @@ The outcome of this is that we will have three Secrets available in the `heptio-
 
 For the purposes of this documentation, we'll be doing the same thing the `contour certgen --pem` or the `make gencerts` commands do and putting the certs in a `certs/` subdirectory of the downloaded repo.
 
+Remember, though, that Contour has `contour certgen --kube` as well - which will do all this for you, and put the results into Kube as Secrets. You can run that locally, or use the Job from [certgen.yaml](../examples/ds-hostnet-split/02-job-certgen.yaml).
```

Comment:
Remember that Contour has a `contour certgen --kube` command which will generate the certificates you need and put the results directly into your cluster.